src/client/updaters/attribute.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Updates the document's html tag attributes
+ *
+ * @param  {Object} attrs - the new document html attributes
+ * @param  {HTMLElement} tag - the HTMLElement tag to update with new attrs
+ */
+export default function updateAttribute({ attribute } = {}, attrs, tag) {
+  const vueMetaAttrString = tag.getAttribute(attribute)
+  const vueMetaAttrs = vueMetaAttrString ? vueMetaAttrString.split(',') : []
+  const toRemove = [].concat(vueMetaAttrs)
+
+  for (const attr in attrs) {
+    if (attrs.hasOwnProperty(attr)) {
+      const val = attrs[attr] || ''
+      tag.setAttribute(attr, val)
+
+      if (vueMetaAttrs.indexOf(attr) === -1) {
```

Comment:
```suggestion
      if (vueMetaAttrs.includes(attr)) {
```

Reply:
Yes, changed this one and a couple of others I missed. Thanks